src/PowerPlots.jl

```diff
@@ -26,9 +26,27 @@ _IM = InfrastructureModels
 const nx = PyCall.PyNULL()
 const scipy = PyCall.PyNULL()
 
+# Create our module level logger (this will get precompiled)
+const _LOGGER = Memento.getlogger(@__MODULE__)
+
+"Suppresses information and warning messages output by PowerPlots, for fine grained control use the Memento package"
+function silence()
+    Memento.info(_LOGGER, "Suppressing information and warning messages for the rest of this session.  Use the Memento package for more fine-grained control of logging.")
+    Memento.setlevel!(Memento.getlogger(PowerPlots), "error")
+end
+
+"alows the user to set the logging level without the need to add Memento"
+function logger_config!(level)
```

Comment:
again, this conflicts with the PowerModels namespace, maybe rename to "PowerPlots_logger_config()" or don't export the function, then `PowerPlots._logger_config()` is required (the module name is required to specify the the namespace)